base/util/PKICertImport.bash

```diff
@@ -159,7 +364,7 @@ function PKICertImport() {
 
     # Show help and usage information.
     function _print_help() {
-        if (( $1 != 0 )); then
+        if (( $1 == 1 )); then
```

Comment:
This helps with spacing so we no longer show an unknown argument error / extra new line when `--help` is specified.